soccer/gameplay/single_robot_sequence.py

```diff
@@ -0,0 +1,65 @@
+import behavior_sequence
+import single_robot_behavior
+import role_assignment
+
+#A sequence to be carried out by a single robot, attempting to add non-single robot behaviors will raise exceptons
+class SingleRobotSequence(behavior_sequence.BehaviorSequence):
```

Comment:
I did a lot of work in single_robot_composite_behavior, so I considered using the same multiple inheritance but honestly I was kind of scared to implement it. I'll give it a go now that I already have a working version